salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -74,7 +74,7 @@ server {
     rewrite '^/Labs$' '/labs' permanent;
```

Comment:
these redirects seem to already preserve the query string:
```
curl -v "https://elifesciences.org/About?p1=v1&p2=v2"
...
< Location: http://elifesciences.org/about?p1=v1&p2=v2
```
```
curl -v "https://elifesciences.org/content/5/e10627?p1=v1&p2=v2"
...
< Location: http://elifesciences.org/articles/10627?p1=v1&p2=v2
```
Are there other scenarios that are broken?